### PR TITLE
Refactor cards

### PR DESCRIPTION
**Pull request by ccabo1**
**Updates**
* Add shade to make selection more intuitive
* Better align margins
* Remove default club image on card view
* Remove unnecessary divs, state, etc.
* Add details at bottom of card
* Kill modal 🍾 (now when you click on the card it goes straight to the club in a new tab)

image

---

**Old stuff (during WIP of this PR)**

image

image

image

image

### CODE REVIEW

**Comment by ccabo1** [comment]
Works well on all screen sizes

**Comment by ccabo1** [comment]
Let's hold off on merging until I kill the modal

File: components/ClubCard.js [hunk 1]
```diff
@@ -72,38 +73,39 @@ const CardTitle = s.strong`
...
 `
 
-class ClubCard extends React.Component {
-  constructor(props) {
-    super(props)
-    this.state = {
-      modal: '',
-    }
-  }
+const shorten = desc => {
```
Comment (ezwang) [inline components/ClubCard.js]: I don't know if it applies in this case, but you could try using [text-overflow: ellipsis](https://www.w3schools.com/cssref/css3_pr_text-overflow.asp) to show exactly the amount of text that can fit inside the box.

Comment (ccabo1) [reply components/ClubCard.js]: Issue with the CSS solution is it only works for one line of text—there's no easy way to have like 3 lines of text where the last line ends in a '...' if needed

Comment (ezwang) [reply components/ClubCard.js]: Looks like [css-line-clamp](https://caniuse.com/#feat=css-line-clamp) is a thing but not supported well on all browsers. Ah well, maybe sometime in the future.

**Pull request merged**